### DISCUSSION TITLE
Artisan graph won't show the power events

### DISCUSSION THREAD

**Discussion by needmorecoffeeroaster** [artisan-roaster-scope/artisan#1759] [Q&A]
when I've been watching training videos on all of the things I can do with artisan plus, they have the power/gas events on the bottom showing.  like a declining staircase, each time the gas/power has been changed.  I cannot for the life of me figure out how to turn this on.  I've tried config, views, etc. and can't seem to get my graph to show these power changes.  Is this due to my Diedrich roaster not putting that info out?  I have a nice roaster and it's only 2 years old, I'm connected to the computer with a USB cord and use Artisan Plus.  Help is appreciated.  Thanks!

**Comment by MAKOMO** [comment]
I am not aware of any training video of artisan.plus, but your issue seems related to the open-source Artisan not artisan.plus (the commercial inventory management service).

Artisan supports a variety of different roasting machines from Diedrich (see https://artisan-scope.org/machines/diedrich/). On that machine page it is also described from which machines event data can be logged. In an case you can enter events manually. There are various options on how to render events controlled under "Markers" in the events dialog (menu Config >> Events, first tab). See also the section on Events in the QuickStartGuide: https://artisan-scope.org/docs/events/
